Dev/src/components/TopBanner.js

```diff
@@ -7,7 +7,7 @@ import { StyleSheet, Text, View, TouchableOpacity } from 'react-native';
 import { Icon } from 'react-native-elements';
```

Comment:
Communicate with Yusuf on which PR to merge first because one of you will have many conflicts